packages/medusa-plugin-ip-lookup/src/api/medusa-middleware.js

```diff
@@ -19,17 +18,20 @@ export default {
         return
       }
 
-      const countryRepo = manager.getCustomRepository(countryRepository)
-      const country = await countryRepo.findOne({
-        where: { iso_2: data.country_code.toLowerCase() },
-      })
+      const lowerCasedCode = data.country_code.toLowerCase()
```

Comment:
Thought: should we normalize within the region service instead?

Reply:
actually, we need both, such that we can attach the lowercased version to the payload